mscore/file.cpp

```diff
@@ -1209,9 +1209,29 @@ QString MuseScore::getFotoFilename(QString& filter, QString* selectedFilter)
       // setup side bar urls
       saveImageDialog->setSidebarUrls(urls);
 
+      // if no saves were made for current score, then use the score's name as default
+      if(cs->masterScore()->savedCapture() == false) {
```

Comment:
We use `if (!cs->masterScore()->savedCapture())`
Notice the space between `if` and `(` and we don't use false